<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInputDlgOutgoingCustomer</class>
 <widget class="QDialog" name="CInputDlgOutgoingCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lava2res.qrc">
    <normaloff>:/images/res/icon.ico</normaloff>:/images/res/icon.ico</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>540</y>
     <width>751</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMask">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>421</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Kommentar:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMaskSet">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>130</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>setzen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditComment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>981</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMask">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vorname</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Strasse</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ort</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Postleitzahl</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kundennummer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kommentar</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBoxAuto">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>130</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Datum:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumDate">
    <date>
     <year>2009</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Kunde:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetCustomer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>981</width>
     <height>131</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Kundennummer</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name, Vorname</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Adresse</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kommentar</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>löschen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Warenliste:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>bearbeiten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNew">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>hinzufügen</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetWares">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>981</width>
     <height>141</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Anzahl</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Artikelbezeichnung</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Artikelnummer(n)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hersteller</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Warengruppe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Standort</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelError">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNomination">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>20</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Vorschlag</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Buchungsnummer:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBookingnumber">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>671</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dateEdit</tabstop>
  <tabstop>lineEditBookingnumber</tabstop>
  <tabstop>pushButtonNomination</tabstop>
  <tabstop>lineEditComment</tabstop>
  <tabstop>comboBoxMask</tabstop>
  <tabstop>lineEditMask</tabstop>
  <tabstop>pushButtonMaskSet</tabstop>
  <tabstop>checkBoxAuto</tabstop>
  <tabstop>tableWidgetCustomer</tabstop>
  <tabstop>pushButtonNew</tabstop>
  <tabstop>pushButtonEdit</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>tableWidgetWares</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="lava2res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CInputDlgOutgoingCustomer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CInputDlgOutgoingCustomer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
